runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsOverlay.java

```diff
@@ -54,6 +54,7 @@
 
 	// Cache of FPS number strings from 00-50
 	private final String[] fpsNums;
+	private static final String FPS_STRING = " FPS";
```

Comment:
Constants belong at the top of the file, before instance variables. Put it below the VALUE_X_OFFSET or something

Reply:
Done